packages/runtime/src/builtin/to_mixed.ts

```diff
@@ -1,13 +1,11 @@
 /* eslint-disable @typescript-eslint/ban-types */
 import {String} from "../types";
 import {ICharacter} from "../types/_character";
-import {INumeric} from "../types/_numeric";
 
 export function to_mixed(input: {
   val: ICharacter | string,
   sep?: ICharacter | string,
-  case?: ICharacter | string,
-  min?: INumeric | number }): String {
```

Comment:
please keep `min` in the code, I know its not used at the moment, but it is a parameter of the method